x-pack/plugins/infra/server/lib/infra_ml/metrics_hosts_anomalies.ts

```diff
@@ -74,24 +61,26 @@ async function getCompatibleAnomaliesJobIds(
 }
 
 export async function getMetricsHostsAnomalies(
-  context: InfraPluginRequestHandlerContext & { infra: Required<InfraRequestHandlerContext> },
+  context: Required<InfraRequestHandlerContext>,
   sourceId: string,
   startTime: number,
   endTime: number,
   metric: 'memory_usage' | 'network_in' | 'network_out' | undefined,
   sort: Sort,
-  pagination: Pagination
+  pagination: Pagination,
+  scoreThreshold: ANOMALY_THRESHOLD = ANOMALY_THRESHOLD.MAJOR,
```

Comment:
now that https://github.com/elastic/kibana/pull/90313 is merged, we can remove the default and fix the ordering of the parameters.  Should probably be an object with such a long list.